src/cuda/image_buffer_cuda.cuh

```diff
@@ -3,7 +3,7 @@
 #include "../image_buffer.h"
 #include "cuda_device.cuh"
 
-namespace PenguinV_Image
+namespace PenguinV
```

Comment:
@RahulRavishankar could you please rename into `penguinv` without capital **P** everywhere?

Reply:
Yes!